flags/conditions.py

```diff
@@ -58,9 +58,7 @@ def get_condition(condition_name):
 def boolean_condition(condition, **kwargs):
     """ Basic boolean check """
     try:
-        if condition.lower() == 'true':
-            return True
-        return False
+        return condition.strip().lower() == 'true'
```

Comment:
I'm not sure I really like the idea of supporting extra spaces here, to be honest. It feels a bit arbitrary. What's the use case, if someone mistypes the string in their settings.py?

If you're going to allow for more flexibility in this value, what about using [`strtobool`](https://docs.python.org/2/distutils/apiref.html#distutils.util.strtobool) that also supports `t`, `y`, etc?

Reply:
Oooh, yes! I honestly didn't know `strtobool` existed! It seems perfect here.

IIRC what originally prompted #27 in the first place was an error where a flag wasn't enabled because an extra space had been entered.

Reply:
Switched to strbool in 4061ab8, but kept the whitespace-stripping.